code-n-color.html

```diff
@@ -0,0 +1,46 @@
+---
+layout: default
+
+title: Code'n'Color
+
+author: Lukas
+content_authors: Code'n'Color
+
+last_update: 2020-06-27
+---
+
+<div class="row pb-2 mb-2">
+
+  <div class="col-md-8">  
+    <h1>Code'n'Color</h1>
+    <h4 class="text-secondary">PhD student support group for Black, Indigenous, and other People of Color (BIPOC)</h4>
+  </div>
+
+  <div class="border-bottom col-md-12">  
+  </div>
+</div>
+
+<div class="row mt-2">
+  <div class="col-md-8">
+    <h2>Description</h2>
+    <p>
+      The support group meets weekly for coffee breaks and hosts BIPOC PhD holders who are leaders in the computing industry and academia. Advised and supported by BIPOC faculty in CS, ECE, and LS, the group is intended to provide a safe and supportive space for young scientists to interact with future faculty peers and climb the treacherous slopes of the ivory tower, cognizant of the unique, different demands and obstacles that BIPOC PhD students face.
+    </p>
+  </div>
+	<div class="col-md-4">
+    
+    <div class="card text-white bg-secondary mb-3" style="max-width: 20rem;">
+      <div class="card-header">Information</div>
+      <div class="card-body">
+          <p class="card-text">Contact: josiah@northwestern.edu</p>
```

Comment:
Maybe point of contact could serve this purpose better

Reply:
Do you mean the title of the card?